cpp/src/parquet/column_reader.h

```diff
@@ -279,6 +279,10 @@ class RecordReader {
   /// \brief Attempt to read indicated number of records from column chunk
   /// \return number of records read
   virtual int64_t ReadRecords(int64_t num_records) = 0;
+  
+  /// \brief Attempt to skip indicated number of records from column chunk
```

Comment:
lets add a detail here to clarify this is the numer of rows records and not Values.

Reply:
Done.